test/bootstrap.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+call_user_func_array(function () {
+    $root = dirname(dirname(__FILE__));
+    $autoload = "$root/vendor/autoload.php";
+
+    if (file_exists($autoload)) {
+        require_once $autoload;
+    }
+}, [__FILE__]);
```

Comment:
Interesting approach. Any reason for enclosing in a function?

Also I think you forgot to consume the argument, or forgot to remove it.

Reply:
It's an [IIFE][]. Here, I use it to avoid polluting the global variable scope.

But yes, I forgot to use the argument. Fixing.

[IIFE]: https://en.wikipedia.org/wiki/Immediately_invoked_function_expression